pyaedt/examples/downloads.py

```diff
@@ -554,6 +554,44 @@ def download_multiparts(destination=None):
     return os.path.join(dest_folder, "library")
 
 
+def download_twin_builder_data(file_name, force_download=False, destination=None):
+    """Download specified Twin Builder example data file.
+
+    Examples files are downloaded to a persistent cache to avoid
+    re-downloading the same file twice.
+
+    Parameters
+    ----------
+    file_name : str
+        Path of the file in the Twin Builder folder.
+    force_download : bool, optional
+        Force to delete file and download file again. Default value is ``False``.
+    destination : str, optional
+        Path where files will be downloaded. Default is ``user temp`` folder.
```

Comment:
Is the name of the folder ``user temp``? If not, we don't want to use literal tags. Based on the example below, It should be "The default is the user's temporary folder." (I see what you don't want to use the pronoun "your" here.)

Reply:
@PipKat This parameter is taken verbatim from other similar functions in this file. 
They mention the user temp folder